TrackletAlgorithm/Constants.h

```diff
@@ -75,7 +158,7 @@ namespace trklet{
   constexpr int N_DISK = 5; // # of endcap disks assumed
 }
 #endif
-constexpr double bfield = 3.8112; // T
+constexpr double bfield = 3.8112; // 
```

Comment:
@aryd Is there a reason for removing the unit in the comment here?

Reply:
Must be a mistake... I have added it back.